src/coordinator.coffee

```diff
@@ -97,9 +97,10 @@ pingUrl = (address, method, callback) ->
 
 class Coordinator extends EventEmitter
   constructor: (@broker, @options = {}) ->
-    @participants = {}
+    @participants = {} # participantId -> Definition (from discovery)
     @connections = {} # connId -> { queue: opt String, handler: opt function }
     @iips = {} # iipId -> { metadata, data }
+    @nodes = {} # role -> { metadata: {} }
```

Comment:
Though, maybe we'll do that in a separate PR where we then make the `nodes` list be what gets dumped to JSON graph instead of participants.